### DISCUSSION TITLE
Memories:  Transcode failed: Transcoder returned 500

### DISCUSSION THREAD

**Discussion by ebildebil** [nextcloud/all-in-one#5332] [Questions] [answered]
Hi,
Memories transcoder suddenly stopped working. (With or without the memories container)
Files play if transcoding is turned off.

Below is the error.

`
/xxxx/nextcloud/extfolders/mount/xxx/androidphone/Internal storage/DCIM/Camera/xxxx.mp4: No such file or directory
2024-09-27T00:16:52.551952767Z 
2024-09-27T00:16:52.551962403Z 2024/09/27 02:16:52 Error creating manager exit status 1
`

Seems to be an issue only with files in external folders.
File/Folder Permissions are: `-rwxrwx--- 1 www-data root`
www-data is member of: `www-data : www-data render`

Transcoding used to work perfectly until a few days ago.
Is there something else I can trouble shoot or can to to fix?

Also marking @pulsejet

**Comment by szaimen** [comment]
Hi, I think that the reason is that https://github.com/nextcloud/all-in-one/blob/0be33a9973fcb2dc5188852f1b0e313db3419a67/php/containers.json#L167-L171 is missing in 
https://github.com/nextcloud/all-in-one/blob/main/community-containers/memories/memories.json

Can you create a PR?

**Reply by ebildebil** [reply]
Hi Simon,
Probably with a lot of learning and training. I'm afarid I have no coding/IT background.

btw: if the above suggestion does fix it, woud that also fix the error when the container is not in use but memories transcoding worked?
I deployed the memories container because I started gettng transcoding errors on a working memories setup without the container.

**Reply by szaimen** [reply, marked as answer]
It will be done with https://github.com/nextcloud/all-in-one/pull/5344

**Reply by ebildebil** [reply]
Thank you.

**Reply by szaimen** [reply]
This is now released with v9.7.0 Beta. Testing and feedback is welcome! See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel
